app/components/navigation/header.tsx

```diff
@@ -9,11 +10,64 @@ import {
   MenuItem,
   IconButton,
   useColorMode,
-  useColorModeValue
+  useColorModeValue,
+  useToast,
+  useBoolean
 } from '@chakra-ui/react'
 import Image from 'next/image'
 import { FiMoon, FiSun } from 'react-icons/fi'
 import { MdAccountCircle, MdLogout } from 'react-icons/md'
+import { signOut } from '@/(auth)/signout/action'
+
+const LogOut = () => {
```

Comment:
In this case, it's better to have LogOut components as an another file so that Header does not care LogOut logic and does not rerendar depending on LogOut component.

Reply:
I agree with Kana's opinion.
If we separate it as another file, Can we create the file in `(auth)/signout/components` or `app/components` ?

Reply:
👍 Moved logout component to `(auth)/signout/components`